cmd/kube-proxy/app/server.go

```diff
@@ -81,6 +81,10 @@ import (
 	"github.com/spf13/pflag"
 )
 
+var (
+	hostNameOverride string
```

Comment:
If we decide to proceed with this one-off approach for hostnameOverride, let's make this an unexported field in `Options`.

Reply:
cc @mtaufen